docs/en/connector-v2/sink/Jdbc.md

```diff
@@ -26,30 +26,33 @@ support `Xa transactions`. You can set `is_exactly_once=true` to enable it.
-| transaction_timeout_sec                   | Int     | No       | -1            |
-| auto_commit                               | Boolean | No       | true          |
-| field_ide                                 | String  | No       | -             |
-| properties                                | Map     | No       | -             |
-| common-options                            |         | no       | -             |
+|                   name                    |  type   | required |        default value         |
+|-------------------------------------------|---------|----------|------------------------------|
+| url                                       | String  | Yes      | -                            |
+| driver                                    | String  | Yes      | -                            |
+| user                                      | String  | No       | -                            |
+| password                                  | String  | No       | -                            |
+| query                                     | String  | No       | -                            |
+| compatible_mode                           | String  | No       | -                            |
+| database                                  | String  | No       | -                            |
+| table                                     | String  | No       | -                            |
+| primary_keys                              | Array   | No       | -                            |
+| support_upsert_by_query_primary_key_exist | Boolean | No       | false                        |
+| connection_check_timeout_sec              | Int     | No       | 30                           |
+| max_retries                               | Int     | No       | 0                            |
+| batch_size                                | Int     | No       | 1000                         |
+| is_exactly_once                           | Boolean | No       | false                        |
+| generate_sink_sql                         | Boolean | No       | false                        |
+| xa_data_source_class_name                 | String  | No       | -                            |
+| max_commit_attempts                       | Int     | No       | 3                            |
+| transaction_timeout_sec                   | Int     | No       | -1                           |
+| auto_commit                               | Boolean | No       | true                         |
+| field_ide                                 | String  | No       | -                            |
+| properties                                | Map     | No       | -                            |
+| common-options                            |         | no       | -                            |
+| schema_save_mode                          | Enum    | no       | CREATE_SCHEMA_WHEN_NOT_EXIST |
+| data_save_mode                            | Enum    | no       | APPEND_DATA                  |
```

Comment:
Can you do a test Demo for this?
for example : https://github.com/apache/seatunnel/blob/e26006d2aa47df0f92740cf9843dca862db82232/docs/en/connector-v2/source/Jdbc.md?plain=1#L246-L260